src/OrbitSshQt/Session.cpp

```diff
@@ -130,6 +130,8 @@ void Session::ConnectToServer(orbit_ssh::Credentials creds) {
 void Session::Disconnect() {
   if (CurrentState() == State::kConnected) {
     SetState(State::kAboutToDisconnect);
+  } else {
```

Comment:
As discussed offline, I would prefer that the handling would be forced on the caller of this function - for example by returning some enum and marking the return value as nodiscard.

Reply:
done